cluster/charts/rook-ceph/values.yaml

```diff
@@ -332,6 +332,9 @@ allowMultipleFilesystems: false
 # For more details see https://github.com/rook/rook/issues/2417
 enableSelinuxRelabeling: true
 
+# The duration between discovering devices in the rook-discover daemonset.
+discoverDevicesInterval: 60m
```

Comment:
this needs to be documented at https://github.com/rook/rook/blob/master/Documentation/helm-operator.md#configuration and do we need to set it by 60m by default?

Reply:
@Madhu-1 The "default" is `60m`, see https://github.com/rook/rook/blob/2bb94441304857870ab81f5fa7b324695f1068f1/pkg/operator/discover/discover.go#L55